### PR TITLE
Rfc 0453 take 2

### PR DESCRIPTION
**Pull request by sklump**
Was https://github.com/hyperledger/aries-cloudagent-python/pull/926

### CODE REVIEW

File: aries_cloudagent/ledger/indy.py [hunk 1]
```diff
@@ -1048,25 +1048,12 @@ async def accept_txn_author_agreement(
...
         if self.pool.cache:
-            # TAA acceptance must be set for each wallet and pool combination
-            cache_key = (
-                TAA_ACCEPTED_RECORD_TYPE
-                + "::"
-                + self.wallet.opened.name
-                + "::"
-                + self.pool.name
-            )
+            cache_key = TAA_ACCEPTED_RECORD_TYPE + "::" + self.pool.name
```
Comment (andrewwhitehead) [inline aries_cloudagent/ledger/indy.py:1051]: I think this is reverting a recent fix for the TAA caching

**Comment by andrewwhitehead** [comment]
Apart from the change to the TAA caching I think it looks good

**Comment by ianco** [comment]
@sklump I'm awaiting on this PR before I release my PR with updates to the alice/faber/performance demo (https://github.com/hyperledger/aries-cloudagent-python/pull/921) so as to not reverse-clobber your demo updates.

File: aries_cloudagent/ledger/indy.py (hunk 1 above)
Comment (sklump) [reply aries_cloudagent/ledger/indy.py:1051]: Do need me to do anything here?

**Comment by sklump** [comment]
> 
> 
> Apart from the change to the TAA caching I think it looks good

I don't understand: 

> 
> 
> Apart from the change to the TAA caching I think it looks good

I don't understand - this is what's in master: https://github.com/hyperledger/aries-cloudagent-python/blob/d3895f6394a5f24a895731bcf6fb672e0d84804e/aries_cloudagent/ledger/indy.py#L1052

I have merged it with master and I haven't touched it. How has it ended up in my PR?

**Comment by sklump** [comment]
> 
> 
> Apart from the change to the TAA caching I think it looks good

To be perfectly clear: you want me to make this look like what's in master?

**Comment by ianco** [comment]
> > Apart from the change to the TAA caching I think it looks good
> 
> To be perfectly clear: you want me to make this look like what's in master?

Yes, just reviewing this the code in master is a recent fix to make TAA work for multitenancy

**Review by ianco — APPROVED** [review]

**Pull request merged**